serverless.ts

```diff
@@ -0,0 +1,44 @@
+import type { AWS } from '@serverless/typescript';
+
+import products from '@functions/productList';
+import productById from '@functions/productById';
+// import hello from '@functions/hello';
+
+const serverlessConfiguration: AWS = {
+  service: 'backend',
+  frameworkVersion: '3',
+  plugins: ['serverless-auto-swagger','serverless-offline' ,'serverless-esbuild'],
+  provider: {
+    name: 'aws',
+    runtime: 'nodejs14.x',
+    apiGateway: {
+      minimumCompressionSize: 1024,
+      shouldStartNameWithService: true,
+    },
+    environment: {
+      AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+      NODE_OPTIONS: '--enable-source-maps --stack-trace-limit=1000',
+    },
+  },
+  // import the function via paths
+  functions: { products, productById },
+  package: { individually: true },
+  custom: {
+    autoswagger:{
```

Comment:
I tried to fetch products by swagger but it doesn't work. 
Try to add `apiType: 'http'` to the autoswagger config, and your swagger will be on the same domain as your endpoints. Furthermore, maybe you need to set `dev` as base path because endpoints are on the `/dev` path (``basePath: '/dev'``)